pkg/common/bitmap/bitmap_test.go

```diff
@@ -121,3 +121,25 @@ func TestC(t *testing.T) {
 	require.True(t, bm5.IsSame(bm5Copy))
 	require.True(t, bm5.Count() == int(bm5Copy.C_Count()))
 }
+
+func TestBitmapIterator_Next(t *testing.T) {
+	np := New(BenchmarkRows)
+	np.AddRange(0, 64)
+
+	// | 63 -- 0 | 127 -- 64 | 191 -- 128 | 255 -- 192 | ... |
+	rows := []uint64{127, 192, 256}
+	np.AddMany(rows)
+
+	itr := np.Iterator()
+	i := uint64(0)
+	for itr.HasNext() {
+		r := itr.Next()
+		if (r < uint64(63)){
+			require.Equal(t, i, r)
+			i++ 
+		}else{
+			fmt.Println("i now is ", i)
```

Comment:
use T.logf() instead of Println

Reply:
sorry and ok, i haven't finished it yet, just wanted to go home and write the rest part.